### PR TITLE
Add settings to optionally simplify structured comments (Doxygen)

### LINKED ISSUE #5706: Leave the non-recognized doxygen commands in comments
It is possible to leave the commands that are not being converted? We use \note, \pre, and \post. The current behavior is that they don't show up in the tool tip at all.

_Originally posted by @jhortenstine in https://github.com/microsoft/vscode-cpptools/issues/658#issuecomment-650328227_

### PR DESCRIPTION
**Pull request by michelleangela**
Add option to simplify structured comments. When the option is set to true, tooltips will display a simplified form of structured comments. For example, for Doxygen, it will only display certain tags. If this option is disabled, it will display complete structured comments.

### Screenshot of setting:
image

### Example displays for Doxygen comment:
image

### Simplified display
image

### Complete display
image

### CODE REVIEW

**Review by sean-mcmanus — APPROVED** [review]

File: Extension/src/nativeStrings.json [hunk 1]
```diff
@@ -189,5 +189,6 @@
         "hint": "This label is for the return value description for a function. Usage example: 'Returns: Area of a shape.'"
     },
     "deprecated_label": "Deprecated:",
-    "exceptions_label": "Exceptions:"
+    "exceptions_label": "Exceptions:",
```
Comment (sean-mcmanus) [inline Extension/src/nativeStrings.json:192]: Are these still used?

Comment (sean-mcmanus) [reply Extension/src/nativeStrings.json:192]: Oh, it is.

Comment (michelleangela) [reply Extension/src/nativeStrings.json:192]: Yes, these labels are used when the comment is simplified.

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]
It seems like the scope should be "application"?

**Comment by michelleangela** [comment]
> It seems like the scope should be "application"?

Since this is UI, then it should only be application? Similar to the experience of changing color theme of VS Code?

**Comment by sean-mcmanus** [comment]
> > It seems like the scope should be "application"?
> 
> Since this is UI, then it should only be application? Similar to the experience of changing color theme of VS Code?

Yeah. If it doesn't make sense for the setting (UI) to change after the user switches workspace folders, switches workspaces, or switches to a remote machine, then the setting should be "application" scope (i.e. User scope).

**Review by sean-mcmanus — APPROVED** [review]

**Pull request merged**